cmd/symbols/ctags-install-alpine.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -eux
+
+# Commit hash of github.com/universal-ctags/ctags
+CTAGS_VERSION=03f933a96d3ef87adbf9d167462d45ce69577edb
+
+apk --no-cache add --virtual \
+    build-deps \
+    curl \
+    jansson-dev \
+    libseccomp-dev \
+    linux-headers \
+    autoconf \
+    pkgconfig \
+    make \
+    automake \
+    gcc \
+    g++ \
+    binutils
```

Comment:
Optional suggestion: I like to keep packages ordered so I can tell more easily if I missed something. As far as I know installation order doesn't matter here (since apk will resolve dependency order anyway).

```suggestion
    autoconf \
    automake \
    binutils \
    build-deps \
    curl \
    g++ \
    gcc \
    jansson-dev \
    libseccomp-dev \
    linux-headers \
    make \
    pkgconfig
```

Reply:
i also follow this practice. But just a note, `build-deps` needs to be the first argument since it is actually attached to the `--virtual` flag (its the name of the virtual list)